vendor/knative.dev/pkg/webhook/certificates/resources/certs.go

```diff
@@ -62,7 +62,7 @@ func createCertTemplate(name, namespace string, notAfter time.Time) (*x509.Certi
 			Organization: []string{organization},
 			CommonName:   commonName,
 		},
-		SignatureAlgorithm:    x509.SHA256WithRSA,
+		SignatureAlgorithm:    x509.PureEd25519,
```

Comment:
This would be a regression, won't work on FIPS enabled clusters.

Reply:
ugh - yeah

related PR here: https://github.com/openshift/knative-eventing/commit/e7b97764a27ebeeb2499796e8f81e4eeca4b882c#diff-37c9e2e0678e8fdad32b5400892c0e3c17f41abd8efcffbe108c8a31e180c0c0